tests/components/cover/test_device_action.py

```diff
@@ -126,7 +126,7 @@ async def test_get_action_capabilities(
     await hass.async_block_till_done()
 
     actions = await async_get_device_automations(hass, "action", device_entry.id)
-    assert len(actions) == 3  # open, close, stop
+    assert len(actions) == 2  # open, close
```

Comment:
Can we pick another entity which supports stop too to not decrease the test coverage?

Reply:
with arguments like this i can not say no. Thanks for the noticing!